### PR TITLE
feat: create install.md and update reame file | add images 

### PR DESCRIPTION
**Pull request by GabrielMazzieiro**

### CODE REVIEW

**Comment by GabrielMazzieiro** [comment]
Readme still needs some updates for the override and publish topics. Please don't 'bother looking them

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+**Prerequisite**: Please follow this [guide](https://code.visualstudio.com/api/working-with-extensions/publishing-extension) to install and login to `vsce`. Ask an existing maintainer how to get the "Personal Access Token".
+
+1. Merge any PR that is ready to be published into `main`.
+2. Perform all the necessary changes following code structure and folder organization.
+3. Update [CHANGELOG.md](https://github.com/404-theme/visual-studio-code/blob/master/CHANGELOG.md) + commit the changes.
+4. Run `vsce publish [version]`. Follow the [SemVer](https://semver.org) convention and replace `[version]` with one of the following options:
+   - `patch` for bug fixes
+   - `minor` for improvements
+   - `major` for breaking or bigger changes
+5. Push the commits and make a [new release](https://github.com/404-theme/visual-studio-code/releases/new).
```
Comment (arthur404dev) [inline README.md]: All this section is related to vscode installation, don't?

We should remove this part from this readme

**Review by arthur404dev — CHANGES_REQUESTED** [review]
Left comments about the end of the readme

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
 <!-- TODO: Utilize same structure as the vscode-theme repo with icon on top and descriptive steps -->
```
Comment (arthur404dev) [inline README.md]: Can we remove this comment aswell? It was only for development

File: README.md (hunk 1 above)
Comment (GabrielMazzieiro) [reply README.md]: topics deleted

**Review by arthur404dev — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (GabrielMazzieiro) [reply README.md]: sure

**Pull request merged**